strongbox-web-core/src/test/resources/etc/conf/strongbox-cron-tasks.xml

```diff
@@ -9,8 +9,6 @@
         <properties>
             <cronExpression>0 0 1 * * ?</cronExpression> <!-- mandatory -->
             <jobClass>org.carlspring.strongbox.cron.jobs.ClearRepositoryTrashCronJob</jobClass> <!-- mandatory -->
-            <!-- <storageId>storage-common-proxies</storageId> optional -->
-            <!-- <repositoryId>maven-central</repositoryId> optional -->
```

Comment:
Why do we still need **strongbox-cron-tasks.xml** for tests ? Shouldn't we drop it ?

Reply:
Yes, I believe we should be able to do so.

Reply:
Done.